deploy/kubernetes/general/apiserver.yaml

```diff
@@ -1,17 +1,51 @@
 ##
 ## Heron API server deployment
 ##
-apiVersion: apps/v1beta1
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    k8s-app: heron-apiserver
+  name: heron-apiserver
+  namespace: default
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: heron-apiserver
+  labels:
+    app: heron-apiserver
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: heron-apiserver
+  namespace: default
+
+---
+
+apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: heron-apiserver
+  labels:
+    app: heron-apiserver
+  namespace: default
 spec:
+  selector:
+    matchLabels:
+      app: heron-apiserver
   replicas: 1
   template:
     metadata:
       labels:
         app: heron-apiserver
     spec:
+      serviceAccountName: heron-apiserver
       containers:
         - name: heron-apiserver
           image: heron/heron:0.16.2
```

Comment:
Can we update the image here too heron/heron:latest?

Reply:
sure